Automattic/MobilePay-Apple
# Remove Alamofire in favor of URLSession

## Description
This PR removes Alamofire in favor of URLSession
- Removed Alamofire from package dependencies
- Added a `Networking` protocol (`URLSession` conforms to this protocol, which enables us to pass`URLSession.shared` as the default parameter for `InAppPurchasesService`

## Notes
- I'm skipping the `InAppPurchaseServiceTests` for now because the `fetchProducts` and `createOrder` methods aren't actually calling `networking.load(request:)` yet

## How to test 

1. Run the iOS app
2. Tap on Rocket Fuel to purchase
3. Tap Confirm
4. Tap OK
5. On Xcode debugger menu, click on the Manage StoreKit Transactions menu item
6. ✅ There should be a record of Rocket Fuel being purchased
